src/Estimators/tests/base-estimator/model.urdf

```diff
@@ -0,0 +1,3411 @@
+<robot name="iCub">
```

Comment:
This question is more for @traversaro. Do you think it is possible to have the model somewhere else?

Reply:
One possibility is to downoad the icub-models repo at configuration time via FetchContent and use the model from there for the tests.

Reply:
While that is a possible solution, we mostly use user-modified models that are rather a bit different from the ones in `icub-models` repo. How do we handle such situations?

Reply:
If the user-modified models are used just for tests, I think we can include a copy here.

Reply:
@GiulioRomualdi how do we proceed?